### PR TITLE
ps: fix module prefix and documentation plus minor cleanup

### PR DESCRIPTION
**Pull request by OlegHahm**
Better ideas for the function name are welcome.

### CODE REVIEW

**Comment by authmillenon** [comment]
Why not just `ps()`. Did the same in `od`.

**Comment by OlegHahm** [comment]
Like this?

**Comment by authmillenon** [comment]
:+1:

File: sys/include/ps.h [hunk 1]
```diff
@@ -26,7 +26,7 @@
 extern "C" {
 #endif
 
-void thread_print_all(void);
+void ps(void);
```
Comment (authmillenon) [inline sys/include/ps.h]: Maybe add some documentation?

File: sys/include/ps.h [hunk 2]
```diff
@@ -26,7 +26,7 @@
...
 int _ps_handler(int argc, char **argv);
```
Comment (authmillenon) [inline sys/include/ps.h]: Any reason whiy this is here?

Comment (authmillenon) [inline sys/include/ps.h]: (I know this is not part of this PR)

**Comment by OlegHahm** [comment]
Updated PR, addressed comments, changed title.

**Comment by authmillenon** [comment]
ACK. Please squash.

**Comment by OlegHahm** [comment]
Squashed.

**Pull request merged**